bot/exts/source.py

```diff
@@ -0,0 +1,100 @@
+import inspect
+from pathlib import Path
+from typing import Optional, Tuple
+
+from discord import Embed
+from discord.ext import commands
+
+from bot.bot import SirRobin
+from bot.constants import URLs
+from bot.converters import SourceConverter, SourceType
+
+
+class BotSource(commands.Cog):
+    """Displays information about the bot's source code."""
+
+    def __init__(self, bot: SirRobin):
+        self.bot = bot
+
+    @commands.command(name="source", aliases=("src",))
+    async def source_command(self, ctx: commands.Context, *, source_item: SourceConverter = None) -> None:
+        """Display information and a GitHub link to the source code of a command, tag, or cog."""
```

Comment:
`tag`s are not in this bot

Reply:
There are multiple places in this file tags are being handled. They could be removed, although for speed it can be ignored.

Please comment this or make a issue as a follow up for this.